suite-common/wallet-utils/src/localizePercentage.ts

```diff
@@ -0,0 +1,18 @@
+export type FormatPercentageArgs = {
+    value: number;
+    locale: string;
+    numDecimals: number;
+};
+
+/**
+ * Format number between 0 and 1 to percentage locale string.
+ */
+export const localizePercentage = ({ value, locale, numDecimals }: FormatPercentageArgs) => {
```

Comment:
nit: how about this, just to be more explicit? Not sure what the default should be though :D 

```suggestion
export const localizePercentage = ({ value, locale, numDecimals = 1 }: FormatPercentageArgs) => {
```